vignettes/analyzing-load-test-logs.Rmd

```diff
@@ -12,31 +12,268 @@ vignette: >
 ```{r setup, include = FALSE}
 knitr::opts_chunk$set(
   collapse = TRUE,
-  comment = "#>"
+  comment = "#>",
+  out.width = "80%",
+  fig.align = "center",
+  fig.width = 8, fig.height = 6
 )
 ```
 
-There are two types of log files generated while using `shinyloadtest`. The first file is the recording log file. This file is a JSON file that captures all of the HTTP(s) and websocket traffic that occured during the recording.
+# Log output
 
-The second file is a session log file generated during the loadtest. A file is generated for each simulated session.
+There are a few types of log files generated by a `shinycannon` recording session.
 
-The two files are tied together by a set of keys that identify unique "events" that were recorded and played back in the load test.
+* `./recording.log`
+  * Copy of the original recording used
+  * This file is a file comprised of JSON lines that capture all of the HTTP(s) and WebSocket traffic that occured during the recording
+* `./detail.log`
+  * Log output to help debug errors
+* `./sessions/*.csv`
+  * Collection of files for each session number, simulated user id, and user session
+  * One file is generated for each simulated session
+  * The file name format displays `SESSION_USER_USERSESSION.csv`
+  * Lines starting with `#` should be ignored when processing
 
+These outputs are combined when creating tidy shinyloadtest data.
 
-Key | Description 
--- | --
-REQ_HOME | The initial HTTP request for the application
-REQ | HTTP requests that represent loading static assets necessary for the app, e.g. javascript or css files.
-WS_SEND | 
-WS_RECV | 
-PLAYER | 
+# Tidy Data
+
+Shiny cannon recording sessions can be loaded into R using the `tidy_loadtest` function.
+
+
+```{r, eval = FALSE}
+df <- shinyloadtest::tidy_loadtest(
+  `Run 1` = "./PATH/TO/RUN_1",
+  `Run 2` = "./PATH/TO/RUN_2",
+  `Run 3` = "./PATH/TO/RUN_3"
+)
+```
 
 The `shinyloadtest::tidy_loadtest` function uses these files to generate a data frame with loadtest results. The resulting data frame contains the following columns:
 
-Column | Description 
--- | --
+* `run`
+  * The name of the recording session. This would be `Run 1` in the example above
+* `session_id`
+  * For the given `run`, an incrementing integer value of the session being recorded. Starts at 0.
+* `user_id`
+  * For the given `run`, which simulated user is performing the work. Starts at 0.
+* `iteration`
+  * An incrementing integer value of the session iteration of a particular `user`. Starts at 0.
```

Comment:
I find session_id vs iteration a bit confusing. Perhaps it'd help to clarify the scope, e.g.

`session_id`: ..., Unique across _all_ sessions.
`iteration`: ..., Unique across this _user's_ sessions.